configs/body_2d_keypoint/topdown_heatmap/mpii/td-hm_hourglass52_8xb32-210e_mpii-384x384.py

```diff
@@ -27,7 +27,7 @@
 auto_scale_lr = dict(base_batch_size=512)
 
 # hooks
-default_hooks = dict(checkpoint=dict(save_best='pck/PCKh', rule='greater'))
+default_hooks = dict(checkpoint=dict(save_best='PCKh', rule='greater'))
```

Comment:
@ly015 How about change the `PCKh` to `PCK` as well?

Reply:
OK